data/hai/hai reveal 4 middle.txt

```diff
@@ -1429,16 +1549,29 @@ mission "Hai Reveal [A11] Sound the Alarm"
 				goto alarm
 			label dinner
+			action
+				set "learned of elders superstition"
 			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the 'Elders' Chambers' - which turns out to be the official name of the whole government complex.`
-			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. I mean, seriously, Tee, what will you do if it doesn't? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
-			`	Teeneep just sips her drink and shakes her head. "Oh, Sayari, you really are getting too long in the tooth for domestic politics these days."`
+			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. Especially after that scene in there; no doubt the long tooths will get riled up."`
+			choice
+				`	"What was that in there about the Mythos?"`
+				`	"What will happen if they get riled up?"`
+					goto riled
+			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a collection of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough there’s a set of people who will believe it’s actually that person come back to steer things.`
+			`	"Someone in there has linked that scene to a story, in which I am implicated as Eyrisen."`
+			label riled
+			`	She scoffs and waves a paw dismissively. "The contingencies shouldn't be affected. Superstition is mostly dead, it's really only the Elders that subscribe to it." She pauses and looks at Sayari. "The plan will work just fine."`
+			`	Sayari grumbles, "There's plenty of older Hai, the long tooths Captain, that subscribe to particular political factions. They tend to also subscribe, at least selectively, to the superstitions. Despite what Teeneep says, I would not count out the potential impact that scene in there could have on the situation."`
+			`	She rounds on Teeneep, "I mean, seriously, Tee, what will you do if the plan doesn't play out fine? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
```

Comment:
```suggestion
			`	She scoffs and waves a paw dismissively. "The contingencies shouldn't be affected. Superstition is mostly dead, it's really only the Elders that believe in it." She pauses and looks at Sayari. "The plan will work just fine."`
			`	Sayari grumbles, "There's plenty of older Hai, those 'long tooths', that align to particular political factions. They also tend to sympathize, at least selectively, to the superstitions. Despite what Teeneep says, I would not count out the potential impact that scene in there could have on the situation."`
			`	She rounds on Teeneep, "I mean, seriously, Tee, what will you do if the plan doesn't play out fine? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
```

Reply:
The change to "align" is fine, but "subscribe" is intentional language for all occurrences referring to the superstition.

Reply:
Yeah, they work, but I think 'subscribe' would be used too much here. 

```suggestion
			`	She scoffs and waves a paw dismissively. "The contingencies shouldn't be affected. Superstition is mostly dead, it's really only the Elders that subscribe to it." She pauses and looks at Sayari. "The plan will work just fine."`
			`	Sayari grumbles, "There's plenty of older Hai, those 'long tooths', that align with particular political factions. They also tend to subscribe, at least selectively, to the superstitions. Despite what Teeneep says, I would not count out the potential impact that scene in there could have on the situation."`
			`	She rounds on Teeneep. "I mean, seriously, Tee, what will you do if the plan doesn't play out fine? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
```
Round means 'to turn'? Then there has to be a full stop if round doesn't meant 'to speak'.